api/seedreconfig/seedreconfig.go

```diff
@@ -67,6 +67,9 @@ type SeedReconfiguration struct {
 	// Example of nmstate configurations can be found in this link https://nmstate.io/examples.html
 	// This field will be used for IBI process as in IBU we will copy nmconnection files
 	RawNMStateConfig string `json:"raw_nm_state_config,omitempty"`
+
+	// The pull secret that obtained from the Pull Secret page on the Red Hat OpenShift Cluster Manager site.
```

Comment:
```suggestion
	// PullSecret is the secret to use when pulling images.
```

(taken from the [installer comment](https://github.com/openshift/installer/blob/31607ba7ad2111ebcaf0d07c5387c211ab3f16b8/pkg/types/installconfig.go#L137))